javascript-data-model/order-history.js

```diff
@@ -0,0 +1,91 @@
+var orderHistory = [
+  {
+    OrderNumber: '114 - 3941689 - 8772232',
+    FinalPrice: 34.00,
+    shipTo: 'JS Masher',
+    item1: {
+      item: 'JavaScript for impatient programmers',
+      author: 'Rauschmayer, Dr. Axel',
+      returnWindow: 'sep, 7, 2020',
+      deliveryDate: 'august, 8, 2020',
+      orderPlacedDate: 'august, 4, 2020',
+      delivered: true,
+      priceOfTheItem: 31.55,
+      eligibleForReturn: true,
+      returned: false,
+      productSupport: false,
+      productReview: null,
+      deliveryToResident: true
+    }
+  },
+  {
+    OrderNumber: '113-9984268-1280257',
+    FinalPrice: 44.53,
+    shipTo: 'JS Masher',
+    item1: {
+      item: 'The Timeless Way of Building',
+      author: 'Alexander, Christopher',
+      returnWindow: 'aug, 19, 2020',
+      deliveryDate: 'July, 20, 2020',
+      orderPlacedDate: 'July, 19, 2020',
+      priceOfTheItem: 41.33,
+      delivered: true,
+      eligibleForReturn: true,
+      returned: false,
+      productSupport: false,
+      productReview: null,
+      deliveryToResident: true
+    }
+  },
+  {
+    OrderNumber: '114-2875557-9059409',
+    FinalPrice: 17.22,
+    shipTo: 'JS Masher',
+    item1: {
+      item: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for Wii U, PC.Support Turbo and Vibration Features.No Driver and No Lag- Gamecube Adapter',
+      author: null,
+      returnWindow: 'aug, 5, 2020',
+      deliveryDate: 'July, 7, 2020',
+      orderPlacedDate: 'July, 4, 2020',
+      priceOfTheItem: 15.98,
+      delivered: true,
+      eligibleForReturn: false,
+      returned: false,
+      productSupport: false,
+      productReview: null,
+      deliveryToResident: true
+    }
+  },
+  {
+    OrderNumber: '113-2883177-2648248',
+    FinalPrice: 138.93,
+    shipTo: 'JS Masher',
+    item1: {
+      item: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+      author: null,
+      returnWindow: 'aug, 4, 2020',
+      priceOfTheItem: 94.95,
+      deliveryDate: 'July, 5, 2020',
+      orderPlacedDate: 'July, 3, 2020',
+      delivered: true,
+      eligibleForReturn: false,
+      returned: false,
+      productSupport: true,
+      productReview: null,
+      deliveryToResident: false
+    },
+    item2: {
+      item: 'The Art of Sql',
+      author: 'Faroult, Stephane',
+      returnWindow: 'aug, 4, 2020',
+      priceOfTheItem: 33.99,
+      deliveryDate: 'July, 5, 2020',
+      orderPlacedDate: 'July, 3, 2020',
+      delivered: true,
+      eligibleForReturn: false,
+      returned: false,
+      productSupport: true,
+      deliveryToResident: false
+    }
```

Comment:
Numbered properties are always bad news. If you find yourself numbering properties, that means you need a list. And if you need a list in javascript, that means you need an `array`